extensions/wrapper/wrapper/src/main/java/de/sovity/edc/ext/wrapper/api/ui/pages/asset/AssetApiService.java

```diff
@@ -34,12 +37,15 @@ public class AssetApiService {
     private final AssetService assetService;
     private final EdcPropertyUtils edcPropertyUtils;
 
-    public List<AssetEntry> getAssets() {
+    public List<UiAsset> getAssets() {
         var assets = getAllAssets();
         return assets.stream().sorted(Comparator.comparing(Asset::getCreatedAt).reversed()).map(asset -> {
-            var entry = new AssetEntry();
-            entry.setProperties(edcPropertyUtils.truncateToMapOfString(asset.getProperties()));
+            var entry = new UiAsset();
+            entry.setAdditionalProperties(edcPropertyUtils.truncateToMapOfString(asset.getProperties()));
             entry.setPrivateProperties(edcPropertyUtils.truncateToMapOfString(asset.getPrivateProperties()));
```

Comment:
These should be an output of the asset mapping process...

Especially the distinction between additional properties and additional json properties